opentelemetry-otlp/src/lib.rs

```diff
@@ -70,10 +69,15 @@
 //! # #[cfg(all(feature = "trace", feature = "grpc-tonic"))]
 //! # {
 //! # fn main() -> Result<(), opentelemetry::trace::TraceError> {
-//! let tracer = opentelemetry_otlp::new_pipeline()
-//!     .tracing()
-//!     .with_exporter(opentelemetry_otlp::new_exporter().tonic())
-//!     .install_batch(opentelemetry_sdk::runtime::AsyncStd)?;
+//! let tracer = opentelemetry_sdk::trace::TracerProvider::builder()
+//!        .with_batch_exporter(
+//!            opentelemetry_otlp::SpanExporter::builder()
+//!                .with_tonic()
+//!                .build()?,
+//!            opentelemetry_sdk::runtime::AsyncStd,
```

Comment:
nit - not related to this PR, but I believe tonic has hard dependency on the tokio runtime. We should be changing this example to use `opentelemetry_sdk::runtime::Tokio`

Reply:
updated